src/main/java/at/ac/fhcampuswien/fhmdb/dao/WatchlistRepository.java

```diff
@@ -12,11 +12,19 @@ public class WatchlistRepository {
     private final Dao<WatchlistEntity, Long> dao;
     private static final String CONNECTION_ERROR_MESSAGE = "Failed to create a connection to the database";
     private static final String MOVIE_ALREADY_IN_THE_WATCHLIST = "Selected movie is already in the watchlist";
+    private static WatchlistRepository instance = null;
```

Comment:
Small thing, `instance` doesn't need to be instantiated to `null` necessarily. You can leave it like this or you can modify it, it's fine either way